python_modules/dagster/README.md

```diff
@@ -42,7 +42,7 @@ from sklearn.linear_model import LinearRegression
 @asset
 def country_populations() -> DataFrame:
     df = read_html("https://tinyurl.com/mry64ebh")[0]
-    df.columns = ["country", "continent", "rg", "pop2018", "pop2019", "change"]
+    df.columns = ["country", "pop2022", "pop2023", "change", "continent", "sr"]
```

Comment:
Let's expand `sr` otherwise it looks like a typo
```suggestion
    df.columns = ["country", "pop2022", "pop2023", "change", "continent", "subregion"]
```

Or 

```suggestion
    df.columns = ["country", "pop2022", "pop2023", "change", "continent", "region"]
```

Hopefully it doesn't make the line too big

Reply:
Good call!